runtime/doc/lsp.txt

```diff
@@ -667,10 +667,29 @@ LspProgress                                                       *LspProgress*
     properties, where `params` is the request params sent by the server (see
     `lsp.ProgressParams`).
 
-    Example: >vim
+    Examples:
+
+    Redraw the statusline whenever an LSP progress message arrives: >vim
         autocmd LspProgress * redrawstatus
 <
 
+    Tell the parent terminal to indicate progress using a native progress
+    indicator (requires a supporting terminal): >lua
```

Comment:
It might be helpful to mention an example or two of supporting terminals (maybe Alacritty, iTerm2, WezTerm?), or a reference to the OSC mechanism used here so that people can look it up for themselves.

I just tried this out in [kitty](https://github.com/kovidgoyal/kitty), only to find that it [added OSC 9 support](https://github.com/kovidgoyal/kitty/issues/1474) in 2019, but later [partially removed it](https://github.com/kovidgoyal/kitty/issues/8011#issuecomment-2452009076) in favor of [its own OSC 99 mechanism](https://github.com/kovidgoyal/kitty/blob/master/docs/desktop-notifications.rst).